db/migrate/20210419171746_add_unique_user_id_index_to_users_settings.rb

```diff
@@ -0,0 +1,7 @@
+class AddUniqueUserIdIndexToUsersSettings < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
```

Comment:
Can we not just remove and add the new index with the unique constraint in one migration file instead of two? 

```
def change:
   remove_index ...
   add_index ...
end
```

Reply:
Rhymes and I attempted this when working on the SQL statement yesterday and it failed 🤷🏾‍♀️  But when we ran the migrations separately, they were successful.

Reply:
Yep! @Ridhwana `strong_migrations` was complaining about something (I don't remember exactly :D). Since they are run in quick succession, I don't think it matters that much. We need it to be a unique index so that `ON CONFLICT DO UPDATE` can work